_posts/2018-09-19-cockpit-178.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Cockpit 178
+author: mvo
+date: 2018-09-19
+tags: cockpit linux
+slug: cockpit-178
+category: release
+summary: No more KubeVirt
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 178.
+
+### Support for KubeVirt has been removed
```

Comment:
### KubeVirt support has been removed

(This drops a word and puts the subject first. It's still passive, but I think that's OK considering the subject is immediate. Otherwise, we'd have to dance around and say "Cockpit has dropped support for KubeVirt" or somesuch... which is a bit redundant due to context.)